graphene_django/filter/filters/list_filter.py

```diff
@@ -1,12 +1,32 @@
+from django_filters.filters import FilterMethod
+
 from .typed_filter import TypedFilter
 
 
+class CustomFilterMethod(FilterMethod):
+    def __call__(self, qs, value):
+        if value is None:
+            return qs
+        return self.method(qs, self.f.field_name, value)
+
+
 class ListFilter(TypedFilter):
     """
     Filter that takes a list of value as input.
     It is for example used for `__in` filters.
     """
 
+    @TypedFilter.method.setter
+    def method(self, value):
+        """
+        Override method setter so that in case a custom `method` is provided, it doesn't fall back to
```

Comment:
The array filter is relying on the `EMPTY_VALUES` check so that's why I didn't add the same implementation. 
I'll still put some tests on it to see if the behaviour of that filter makes sense

Reply:
While adding tests I realized that none of the array field tests were getting run because we were not having postgres in the test environment, so I ended up revamping all of those tests to fake an `ArrayField` with a `JSONField` instead for unit tests only.

And you are correct that the same logic was needed in the `ArrayFilter`